api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoTrimestralCohortQueries.java

```diff
@@ -42,18 +42,43 @@ public ResumoTrimestralCohortQueries(
     this.hivMetadata = hivMetadata;
   }
 
+  /**
+   * Number of patients transferred-in from another HFs during the current month
+   *
+   * @return CohortDefinition
+   */
+  public CohortDefinition
```

Comment:
All methods should be private by default. Please change to private end move to end of the class.

Reply:
Done